George/less/footer.less

```diff
@@ -0,0 +1,25 @@
+.footer {
+  
+
+  .Footer-Background {
```

Comment:
Ideally, there wouldn't be any white-space above this class here. If you want to use something that'll help with uniform code formatting, try out [prettier](https://prettier.io). It's available for a bunch of code editors and is pretty easy to use. Using a tool for code formatting is also common when working with other developers, so it's great to go ahead and get some experience with one of them.